website/templates/emails/email_contributors_of_registrations.txt.mako

```diff
@@ -0,0 +1,18 @@
+Dear ${user.fullname}, 
+
+You are receiving this message because you have an account on the Open Science Framework (OSF).  We are writing to let you know of a change to our model for registrations that may affect how you choose to use that feature.
```

Comment:
@bnosek @sloria Need to update this with however we want to work the tentative dates